doc/nrf/app_dev/device_guides/nrf91/nrf91_snippet.rst

```diff
@@ -83,6 +83,20 @@ This can be done in one of the following ways:
 
       See :ref:`cmake_options` for more details.
 
+.. _nrf91_modem_trace_rtt_snippet:
```

Comment:
The table here : https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/app_dev/device_guides/nrf91/nrf91_snippet.html also would require an update.